server.js

```diff
@@ -8,7 +8,7 @@ const requestProxy = require('express-request-proxy'); // REVIEW: We've added a
 const PORT = process.env.PORT || 3000;
 const app = express();
 // const conString = 'postgres://USERNAME:PASSWORD@HOST:PORT';
-const conString = ''; // TODO: Don't forget to set your own conString
+const conString = 'postgres://USERNAME:PASSWORD@HOST:PORT'; // TODO: Don't forget to set your own conString
```

Comment:
I thought you were told to set environment variable in Heroku and set constring as `process.env.DATABASE_URL` for this class?